src/main/scala/cromwell/engine/CallExecutionActor.scala

```diff
@@ -22,7 +23,8 @@ class CallExecutionActor(backendCall: BackendCall) extends Actor with CromwellAc
 
   private def execute(): Unit = {
     log.info(s"$tag: starting ${backendCall.call.name} for workflow ${backendCall.workflowDescriptor.shortId}")
-    val result = backendCall.execute
+    // If the actual execution throws an exception, catch and wrap with a FailedExecution (DSDEEPB-1579).
```

Comment:
re having the jira ticket in the comment. @mcovarr and I discussed the relative pros and cons of this in the context of cromwell being a public repo. Our decision was as a general rule we will not use JIRA references but if a particular case seems really important to have it we'll debate the individual case on its merits